npm install --prefix assets error on windows 10

\Projects\elixir\phoenix>elixir -v
Erlang/OTP 21 [erts-10.3] [64-bit] [smp:4:4] [ds:4:4:10] [async-threads:1]

Elixir 1.10.2 (compiled with Erlang/OTP 21)

\Projects\elixir\phoenix>mix phx.new -v
Phoenix v1.4.15

\Projects\elixir\phoenix>node -v
v12.16.1

after cloning and mix deps.get successfully I attempt 
npm install --prefix assets 
and  I get 
\Projects\elixir\phoenix\phoenix-liveview-counter-tutorial>npm install --prefix assets
npm ERR! code ENOLOCAL
npm ERR! Could not install from "" as it does not contain a package.json file.

npm ERR! A complete log of this run can be found in:
npm ERR!     
\AppData\Roaming\npm-cache\_logs\2020-03-12T16_40_00_067Z-debug.log

[2020-03-12T16_40_00_067Z-debug.log](https://github.com/dwyl/phoenix-liveview-counter-tutorial/files/4325431/2020-03-12T16_40_00_067Z-debug.log)

--- a/README.md
+++ b/README.md
@@ -200,6 +200,7 @@ Install the `Node.js` dependencies with:
 
 ```sh
 npm install --prefix assets
+# or `cd assets && npm install && cd ..` for Windows users if --prefix doesn't work
 ```
 
 It will take a few seconds to download the dependencies
@@ -667,6 +668,7 @@ you will need to run the following command to _install_ the dependency:
 
 ```sh
 npm install --prefix assets
+# or `cd assets && npm install && cd ..` for Windows users if --prefix doesn't work
 ```
 
 You should expect to see output similar to the following:

